app/views/settings/dex_connector_ldaps/_form.html.slim

```diff
@@ -0,0 +1,138 @@
+= form_for [:settings, @certificate_holder], html: { class: "dex-connectors-form"} do |f|
+  .form-group class="#{error_class_for(@certificate_holder, :name)}"
+    = f.label :name
+    = f.text_field :name, class: "form-control", value: @certificate_holder.name
+    = error_messages_for(@certificate_holder, :name)
+
+  hr
+  h4 Server
+
+  .form-group class="#{error_class_for(@certificate_holder, :host)}"
+    = f.label :host
+    = f.text_field :host, class: "form-control", value: @certificate_holder.host
+    = error_messages_for(@certificate_holder, :host)
+
+  .form-group class="#{error_class_for(@certificate_holder, :port)}"
+    = f.label :port
+    = f.text_field :port, class: "form-control", value: @certificate_holder.port
+    = error_messages_for(@certificate_holder, :port)
+
+  .form-group class="#{error_class_for(@certificate_holder, :start_tls)}"
+    = f.label :start_tls, "StartTLS"
+    br
+    .btn-group.btn-group-toggle data-toggle="buttons"
+      = f.label :start_tls, nil, class: "btn btn-default #{'btn-primary active' if @certificate_holder.start_tls}"
+        = f.radio_button :start_tls, "true", checked: @certificate_holder.start_tls
+        | On
+      = f.label :start_tls, nil, class: "btn btn-default #{'btn-primary active' unless @certificate_holder.start_tls}"
+        = f.radio_button :start_tls, "false", checked: !@certificate_holder.start_tls
+        | Off
+    = error_messages_for(@certificate_holder, :start_tls)
+
+
+  .form-group class="#{error_class_for(@certificate_holder, :certificate)}"
+    = f.label :certificate
+    p Paste the certificate for the LDAP server here.
+    = f.text_area :certificate, value: @cert.certificate, class: "form-control", rows: 5, required: @certificate_holder.start_tls
```

Comment:
We are handling certificate as file upload now so I guess it makes sense to go with that here too. 

You can check either https://github.com/kubic-project/velum/pull/593 or https://github.com/kubic-project/velum/pull/590 to get some inspiration on how to solve this.